docs/hooks/useTransactor.md

```diff
@@ -0,0 +1,72 @@
+---
+sidebar_position: 7
+---
+
+# useTransactor
+
+Use this hook to interact with the chain and get UI feedback on the transaction status. You can pass in anything that is a valid parameter to [Viem's `sendTransaction` function](https://viem.sh/docs/actions/wallet/sendTransaction#parameters).
+
+```ts
+const transactor = useTransactor();
+const writeTx = transactor({
+  to: '0x70997970c51812dc3a010c7d01b50e0d17dc79c8',
+  value: 1000000000000000000n,
+});
+await writeTx();
+```
+
+This example tries to send 1 ETH to the address `0x70997970c51812dc3a010c7d01b50e0d17dc79c8`, prompting the connected `WalletClient` for a signature. And in the case of a successful transaction, it will show a popup in the UI with the message: "🎉 Transaction completed successfully!".
```

Comment:
```suggestion
This example tries to send 1 ETH to the address `0x70997970c51812dc3a010c7d01b50e0d17dc79c8`, prompting the connected [`WalletClient`](https://wagmi.sh/react/api/hooks/useWalletClient#usewalletclient) for a signature. And in the case of a successful transaction, it will show a popup in the UI with the message: "🎉 Transaction completed successfully!".
```

Reply:
https://github.com/scaffold-eth/se-2-docs/commit/aee53a3ed5465b9785e3b9fbbf00d03e1f8f3832